standalone/README.md

```diff
@@ -55,6 +55,8 @@ These instructions are to be followed in the `standalone` directory of this repo
 - Run `make init`
 - Run `make all` to setup simple-server on your servers.
     - Simple server should now be installed, running and accessible on your domain.
+    - Note: Some versions of MacOS fail on running the prometheus setup scripts. You will have to run
```

Comment:
If you still have it, can you add some detail on the specific error that this will fix?

Reply:
Added to note.